symengine/dict.h

```diff
@@ -42,6 +43,7 @@ typedef std::map<RCP<const Integer>, unsigned,
         RCPIntegerKeyLess> map_integer_uint;
 typedef std::map<unsigned, integer_class>
        map_uint_mpz;
+typedef std::map<unsigned, Expression> map_uint_Expr;
```

Comment:
@isuruf Same question ^.  Are signed degrees somewhere required in Series expansions ?